services/src/main/resources/locale/portlet/news/News_fr.properties

```diff
@@ -191,7 +191,7 @@ news.title.confirmDeleteDraftNews=Supprimer le brouillon?
 news.button.ok=Valider
 news.button.cancel=Annuler
 news.share.message=L'article a \u00E9t\u00E9 partag\u00E9 dans
-news.illustration.link.title=Ouvrir l'article
+news.illustration.link.title=Ouvrir l'article {0}
```

Comment:
fr not needed IMO, it should be handled by crowdin
WDYT @mkrout ?

Reply:
we should let it for test, it will be erased by crowdin